crates/tx5-go-pion-sys/build.rs

```diff
@@ -155,6 +155,29 @@ fn go_build_cmd(
 
     cmd.env("CGO_ENABLED", "1");
 
+    let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap();
+    if target_os == "ios" {
```

Comment:
Can you use the already determined TARGET.go_os?

```suggestion
    if TARGET.go_os == "ios" {
```